File: app/src/main/java/com/mss/instamat/domain/repositories/FilesRepository.java
package com.mss.instamat.domain.repositories;

import android.graphics.Bitmap;
import android.support.annotation.NonNull;

import com.mss.instamat.domain.models.ImageInfo;

import java.io.IOException;

public interface FilesRepository {

    @NonNull
    String saveBitmap(@NonNull final ImageInfo imageInfo, @NonNull final Bitmap bitmap) throws IOException;
}

File: app/src/main/java/com/mss/imagesearcher/model/repositories/FilesRepository.kt
package com.mss.imagesearcher.model.repositories

import android.graphics.Bitmap

import com.mss.imagesearcher.model.entity.ImageInfo

import java.io.IOException

interface FilesRepository {

    @Throws(IOException::class)
    fun saveBitmap(imageInfo: ImageInfo, bitmap: Bitmap): String
}
